nav2_controller/src/controller_server.cpp

```diff
@@ -265,6 +268,15 @@ ControllerServer::on_activate(const rclcpp_lifecycle::State & /*state*/)
   ControllerMap::iterator it;
   for (it = controllers_.begin(); it != controllers_.end(); ++it) {
     it->second->activate();
```

Comment:
I think you forgot to remove this line

Also, can `activate` throw? If not, then we can remove this code change